src/config.xml

```diff
@@ -3933,5 +3956,7 @@ This setting is not only used for dot files but also for msc temporary files.
     <option type='obsolete' orgtype='bool' id='RTF_SOURCE_CODE'/>
     <option type='obsolete' orgtype='bool' id='LATEX_SOURCE_CODE'/>
     <option type='obsolete' orgtype='bool' id='CLASS_DIAGRAMS'/>
+    <option type='obsolete' orgtype='int' id='DOT_FONTSIZE'/>
+    <option type='obsolete' orgtype='string' id='DOT_FONTNAME'/>
```

Comment:
Some compatibility code is necessary in `configimpl.l` so that users that did set `DOT_FONTSIZE` or `DOT_FONTNAME` will still see these values. See function `void Config::updateObsolete()` in `configimpl.l`.
(Probably also the tests for `FreeSans` etc should be used here.

Reply:
Removed obsolete and added conversion in updateObsolete

Reply:
The settings `DOT_FONTSIZE` and `DOT_FONTNAME` should still be set to `obsolete` as otherwise they will still be in the the `Doxyfile`